java/jdbc/src/main/java/io/vitess/jdbc/VitessVTGateManager.java

```diff
@@ -125,11 +127,14 @@ private static VTGateConn getVtGateConn(VitessJDBCUrl.HostInfo hostInfo, VitessC
                     ? connection.getTrustStore() : System.getProperty(Constants.Property.TRUSTSTORE_FULL);
             final String trustStorePassword = connection.getTrustStorePassword() != null
                     ? connection.getTrustStorePassword() : System.getProperty(Constants.Property.TRUSTSTORE_PASSWORD_FULL);
+            final String trustStoreType = connection.getTrustStoreType() != null
```

Comment:
I think you need to add this to TlsOptions below?

Reply:
yes!